app/src/main/java/com/example/concordia_campus_guide/Database/Converters/CoordinatesListConverter.java

```diff
@@ -22,18 +23,16 @@ public String convertToDatabaseColumn(List<Coordinates> listOfCoordinates) {
         }
         Gson gson = new Gson();
         Type type = new TypeToken<List<Coordinates>>() {}.getType();
-        String json = gson.toJson(listOfCoordinates, type);
-        return json;
+        return gson.toJson(listOfCoordinates, type);
     }
 
     @TypeConverter
     public List<Coordinates> convertToEntityAttribute(String listOfCoordinatesString) {
         if (listOfCoordinatesString == null){
-            return null;
+            return new ArrayList<>();
```

Comment:
did you change the place where this was being used to check for .empty() instead of null?

Reply:
I think you did @CarenRizk remember I was telling you some tests were failing b/c of that and you fixed them right?

Reply:
It is inside the typeConverter so it should be fine